custom_components/tuya_local/devices/fresco_hydrateultra_petfountain_alt.yaml

```diff
@@ -0,0 +1,233 @@
+    class: duration
+    category: diagnostic
+    dps:
+      - id: 15
+        type: integer
+        name: sensor
+        unit: s
+
+  - entity: sensor
+    name: Last Water change
+    icon: "mdi:water-circle"
+    category: diagnostic
+    class: duration
+    dps:
+      - id: 101
+        type: integer
+        name: sensor
+        range:
+          min: 0
+          max: 7200
+        unit: min
+
+  - entity: sensor
+    name: Water health
+    icon: "mdi:water-circle"
+    category: diagnostic
+    class: duration
+    dps:
+      - id: 101
+        type: integer
+        name: sensor
+        range:
+          min: 0
+          max: 7200
+        mapping:
+          - target_range:
+              min: 100
+              max: 0
+        precision: 1
+        unit: "%"
+
+  - entity: button
+    name: Reset Water health
+    icon: "mdi:sync"
+    category: config
+    dps:
+      - id: 101
+        type: integer
+        name: button
+        mapping:
+          - dps_val: 0
+            value: true
+
+  - entity: sensor
+    name: Last Filter change
+    icon: "mdi:air-filter"
+    category: diagnostic
+    class: duration
+    dps:
+      - id: 102
+        type: integer
+        name: sensor
+        range:
+          min: 0
+          max: 43200
+        unit: "m"
+
+  - entity: sensor
+    name: Filter health
+    icon: "mdi:air-filter"
+    category: diagnostic
+    class: duration
+    dps:
+      - id: 102
+        type: integer
+        name: sensor
+        range:
+          min: 0
+          max: 43200
+        mapping:
+          - target_range:
+              min: 100
+              max: 0
+        precision: 1
+        unit: "%"
+
+
+  - entity: button
+    name: Reset Filter health
+    icon: "mdi:sync"
+    category: config
+    dps:
+      - id: 102
+        type: integer
+        name: button
+        mapping:
+          - dps_val: 0
+            value: true
+
+  - entity: sensor
+    name: Last pump maintenance
+    icon: "mdi:pump"
+    class: duration
+    category: diagnostic
+    dps:
+      - id: 103
+        type: integer
+        name: sensor
+        range:
+          min: 0
+          max: 86400
+        unit: min
+
+  - entity: sensor
+    name: Pump health
+    icon: "mdi:pump"
+    class: duration
+    category: diagnostic
+    dps:
+      - id: 103
+        type: integer
+        name: sensor
+        range:
+          min: 0
+          max: 86400
+        mapping:
+          - target_range:
+              min: 100
+              max: 0
+        precision: 1
+        unit: "%"
+
+  - entity: button
+    name: Reset Pump health
+    icon: "mdi:sync"
+    category: config
+    dps:
+      - id: 103
+        type: integer
+        name: button
+        mapping:
+          - dps_val: 0
+            value: true
+
+  - entity: sensor
+    name: Working Time
+    icon: mdi:briefcase-clock
+    category: diagnostic
+    dps:
+      - id: 104
+        type: integer
+        name: sensor
+        range:
```

Comment:
You don't need a range on a read-only value, unless you are inverting it or shifting it into a different target range.

Suggest adding a unit and `class: duration`.

And maybe a more standard name is "Runtime".

Reply:
This is a bit a tricky one since it is technically r/w but the format is not self explanatory. As per the docs:

> "The highest digit (0/1) indicates rule enable. The following 8 digits indicate start and end time. 20102320 means 20:10~23:20"

So it doesn't really have a unit. How would you deal with this? Is the `class: duration` neccessary and fitting?